README.md

```diff
@@ -5,6 +5,68 @@ https://flutter.dev/
 Flutter in an open Source SDK for creating high-performance mobile apps for IOS and Android.
 The Flutter makes it easier for you to build user interfaces, while reducing the amount of code required to create and update your app.
 
+## Why use Flutter?
+
+- Flutter can be used to build applications for Android and IOS
+- The Dart language used in Flutter is quite similar to Java
+- We can update the application in real time
+- If we close the application when we open it again we can continue from where we were
+- Flutter benefits from several objects that help us to facilitate the construction process
+
+## Core Principles
+
+Flutter is designed for the creation of 2D mobile applications.</br> 
```

Comment:
This is true. But you don't need to say "2D" because if it's not "3D" then "2D" is assumed. 
Also there are people who have built 3D Widgets in Unity and used the in Flutter apps,
so it's 100% possible to build a 3D game with flutter, it's just not the _norm_ (_yet_).
see: https://hackernoon.com/what-i-learned-writing-game-from-scratch-with-flutter-ls82g31rj